contents/blog/decouple-deployment-from-release.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Why (and how) you should decouple deployment from release
+date: 2023-09-07
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+You build software for others to use. This means you need a process for getting that software to those people. This is easier than ever thanks to the internet, hosting services, code management platforms, integration pipelines, and more innovations.
+
+What remains a challenge is delivering code to people **safely**. A best practice for doing this is decoupling deployment and release, but what does this even mean? This post goes over the differences between the two, why you should decouple them, examples, and how to get started.
```

Comment:
The first paragraph is a bit dull, maybe we can remove it?
Maybe something like this:

```suggestion
Picture this: You and your team have been working for months on a large feature. The time has come to release the feature, but how can you do so **safely** and without causing any outages or major issue?

A best practice for doing this is decoupling deployment and release. What does this mean, though? This post goes over the differences between the two, why you should decouple them, examples, and how to get started.
```

Reply:
Rewrote